src/main/java/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -321,10 +324,11 @@ public void onConfigurationChanged(Configuration newConfig) {
     composeText.setTransport(sendButton.getSelectedTransport());
     quickAttachmentDrawer.onConfigurationChanged();
 
-    if (emojiDrawerStub.resolved() && container.getCurrentInput() == emojiDrawerStub.get()) {
+    if (emojiPicker != null && container.getCurrentInput() == emojiPicker) {
       container.hideAttachedInput(true);
     }
 
+    reloadEmojiPicker();
```

Comment:
potential optimization: keep track of current orientation in a variable and only call this if orientation `!=` previous saved orientation, to avoid reloading in other kind of configuration changes